pipeline/authn/authenticator_oauth2_introspection.go

```diff
@@ -27,6 +27,7 @@ type AuthenticatorOAuth2IntrospectionConfiguration struct {
 	ScopeStrategy       string                                                `json:"scope_strategy"`
 	IntrospectionURL    string                                                `json:"introspection_url"`
 	BearerTokenLocation *helper.BearerTokenLocation                           `json:"token_from"`
+	RequestHeaders      map[string]string                                     `json:"request_headers"`
```

Comment:
Maybe `introspection_request_headers` to make it explicit that this is for the introspection request?

Reply:
makes sense